browser/ui/tabs/brave_tab_color_mixer.cc

```diff
@@ -121,6 +125,9 @@ void AddBraveTabDarkThemeColorMixer(ui::ColorProvider* provider,
       mixer[color_id] = color;
     }
   }
+
+  mixer[kColorBraveSplitViewInactiveWebViewBorder] = {
```

Comment:
We really need to plug this in so you don't have to set the light and the dark color manually :cry:

Reply:
Yeah, I totally agree. We're losing benefit of the design system at least in the C++ code.